src/operator/numpy/np_elemwise_binary_gcd_op.cc

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*!
+ *  Copyright (c) 2019 by Contributors
+ * \file np_elemwise_binary_gcd_op.cc
+ * \brief CPU Implementation of greatest common divisor.
+ */
+
+#include "np_elemwise_binary_gcd_op-inl.h"
+#include <string>
+#include <utility>
+
+namespace mxnet {
+namespace op {
+
+NNVM_REGISTER_OP(_np_gcd)
```

Comment:
This function needs to be registered as `_npi_gcd` and wrap it in python to support input as a Python scalar value. Please refer to `mxnet.numpy.add` for similar implementation.
https://github.com/apache/incubator-mxnet/blob/numpy/python/mxnet/ndarray/numpy/_op.py#L312
https://github.com/apache/incubator-mxnet/blob/numpy/python/mxnet/numpy/multiarray.py#L1517